src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.ts

```diff
@@ -16,6 +16,7 @@ import { ModalService } from '~/app/shared/services/modal.service';
 export class LoginComponent implements OnInit {
   model = new Credentials();
   isLoginActive = false;
+  isCreateCluster = false;
```

Comment:
For consistency with the dashboard setting name, wouldn't be better to name it
``ìnitCreateCluster```?

Reply:
Done